lib/tasks/set_taxonomy_topic_email_override.rake

```diff
@@ -0,0 +1,25 @@
+desc "override email notifications for a document collection to a given taxonomy topic, can only be run on against a document that has never been published."
+task :set_taxonomy_topic_email_override, %i[document_collection_id taxon_content_id confirmation_string] => :environment do |_, args|
```

Comment:
I'd say the general convention for the `confirmation_string` argument would be to call it `dry_run`. This convention is already present in Whitehall. The [change note remover](https://github.com/alphagov/whitehall/blob/39df09dd26544ef5989dd162e1fa1bc7c123e72f/lib/tasks/data_hygiene.rake#L5) makes use of it

Reply:
Thanks - there was convo about this [here](https://github.com/alphagov/whitehall/pull/8986#pullrequestreview-2028137615) and in an earlier version Becca had two rake tasks (dry run and for reals).

Shall I go back to `dry_run` with a boolean? As app owner I think you get casting vote!

Reply:
Haha, I hadn't read that. @unoduetre is right about it typically being true by default. Probably the nicest thing would be a confirmation step so the operator would have to type "y" or similar to proceed. Quite a few CLI libraries make that fairly easy but Rake does not, as far as I can see, so probably stick with what you have for now. I don't feel strongly about it.